core/tests/protractor_desktop/editorFeatures.js

```diff
@@ -917,7 +917,7 @@ describe('Exploration translation', function() {
     explorationEditorTranslationTab = explorationEditorPage.getTranslationTab();
   });
 
-  it('should have all the state contents', function() {
+  it('should have all the state contents with numerical status', function() {
```

Comment:
Can you please create different tests for this (As in future we can add audio to different states and check the status value.)?

Reply:
Sure

Reply:
Done